packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.ts

```diff
@@ -35,16 +35,15 @@ export function trackViewMetrics(
     scheduleViewUpdate()
   })
 
-  const { setActivityLoadingTime, setLoadEvent } = trackLoadingTime(loadingType, (newLoadingTime) => {
-    viewMetrics.loadingTime = newLoadingTime
-    scheduleViewUpdate()
-  })
-
-  const { stop: stopActivityLoadingTimeTracking } = trackActivityLoadingTime(
+  const { stopWaitIdlePage, setLoadEvent } = trackLoadingTime(
```

Comment:
**💬 suggestion:** no need to explicitly expose internal component stop method
```suggestion
  const { stop: stopLoadingTimeTracking, setLoadEvent } = trackLoadingTime(
```

Reply:
Yes, sure